spmd/compiler/fusion.py

```diff
@@ -802,29 +775,23 @@ def run_overlap_communication(gm: fx.GraphModule) -> None:
         graph_info, gm, comm_type=CommType.ALLREDUCE
     )  # type:ignore[arg-type]
 
-    _debug(f"length of fe_list {len(fe_list)}")  # type: ignore
+    logger.debug(f"length of fe_list {len(fe_list)}")  # type: ignore
```

Comment:
any reason to use debug instead of info here? do we have a rule when to use which?

Reply:
This is useful most likely for debugging hence the change.